Pipfile

```diff
@@ -69,6 +69,8 @@ pycountry = ">=19.7.15"
 xmltodict = "*"
 # TODO: to be removed when the thumbnail will be refactored
 angular-gettext-babel= ">=0.1"
+# Avoid CVE 38076 on bleach <=3.1.1 ( asked by invenio-record-rest)
```

Comment:
I would write something like: dependency of invenio-record-reset to avoid confusion.

Reply:
You're right, I change it!